### PR TITLE
Add a text splitter component

### PR DESCRIPTION
**Pull request by alimosaed**
**What is the purpose of this change?**

The vector database ingestion gets a list of chunked texts; however, we need to ingest a document which includes a long text. 

**How is this accomplished?**
Added a text splitter component using Langchain to get a text and split it to chunks.

**Anything reviews should focus on/be aware of?**

### CODE REVIEW

File: examples/llm/openai_chroma_rag.yaml
Comment (cyrus2281) [inline examples/llm/openai_chroma_rag.yaml:1]: This example is highly used in our demos, can we create another example/update another one

Comment (cyrus2281) [reply examples/llm/openai_chroma_rag.yaml:1]: Or use the same example, just add a new flow.
Like one flow for paragraph input, and one flow for array of text input

**Review by cyrus2281 — APPROVED** [review]
The code looks good.

Added one request for the example

File: examples/llm/openai_chroma_rag.yaml
Comment (alimosaed) [reply examples/llm/openai_chroma_rag.yaml:1]: @cyrus2281 I returned the original example.

**Pull request merged**